KomaMRIBase/src/datatypes/Sequence.jl

```diff
@@ -170,13 +170,13 @@ end
 
 """
     y = is_RF_on(x::Sequence)
-    y = is_RF_on(x::Sequence, t::Vector{Float64})
+    y = is_RF_on(x::Sequence, t::Vector{Real})
 
```

Comment:
rename `x` by `seq`

Reply:
Addressed in [949850c](https://github.com/cncastillo/KomaMRI.jl/pull/297/commits/949850cbe8affdd01f1705d53c7323d756012ec3)